test/engine/info-plugin.ts

```diff
@@ -0,0 +1,314 @@
+import infoPlugin, { engineInfo } from '../../src/engine/info-plugin'
+
+describe('Engine info plugin', () => {
+  // Helper to create mock heading tokens
+  const createHeadingTokens = (tag: string, content: string) => [
```

Comment:
It would be better to test with the result of parse by actual Marp or markdown-it instance.